diff --git a/src/App.tsx b/src/App.tsx
index 88df2dd..8545638 100644
--- a/src/App.tsx
+++ b/src/App.tsx
@@ -15,6 +15,11 @@ import PaginaListaFuncionarios from './pages/funcionarios/PaginaListaFuncionario
 import PaginaFormularioFuncionario from './pages/funcionarios/PaginaFormularioFuncionario';
 import PaginaChat from './pages/chat/PaginaChat';
 import PaginaFinanceiro from './pages/financeiro/PaginaFinanceiro';
+import PaginaListaPedidos from './pages/pedidos/PaginaListaPedidos';
+import PaginaDetalhePedido from './pages/pedidos/PaginaDetalhePedido';
+import PaginaInformacaoLoja from './pages/configuracoes/PaginaInformacaoLoja';
+import PaginaEditarInfoLoja from './pages/configuracoes/PaginaEditarInfoLoja';
+import PaginaTaxaEntrega from './pages/configuracoes/PaginaTaxaEntrega';
 
 function App() {
   return (
@@ -86,6 +91,48 @@ function App() {
             }
           />
 
+          <Route
+            path="/pedidos"
+            element={
+              <RotaProtegida cargosPermitidos={['administrador', 'gerente', 'atendente']}>
+                <PaginaListaPedidos />
+              </RotaProtegida>
+            }
+          />
+          <Route
+            path="/pedidos/:id"
+            element={
+              <RotaProtegida cargosPermitidos={['administrador', 'gerente', 'atendente']}>
+                <PaginaDetalhePedido />
+              </RotaProtegida>
+            }
+          />
+
+          <Route
+            path="/configuracoes"
+            element={
+              <RotaProtegida cargosPermitidos={['administrador']}>
+                <PaginaInformacaoLoja />
+              </RotaProtegida>
+            }
+          />
+          <Route
+            path="/configuracoes/editar"
+            element={
+              <RotaProtegida cargosPermitidos={['administrador']}>
+                <PaginaEditarInfoLoja />
+              </RotaProtegida>
+            }
+          />
+          <Route
+            path="/configuracoes/taxa-entrega"
+            element={
+              <RotaProtegida cargosPermitidos={['administrador']}>
+                <PaginaTaxaEntrega />
+              </RotaProtegida>
+            }
+          />
+
           <Route
             path="/financeiro"
             element={
